### PR TITLE
Document example to cast MongoDB ObjectId to VARCHAR

### PR DESCRIPTION
**Pull request by blcksrx**

### CODE REVIEW

File: presto-docs/src/main/sphinx/connector/mongodb.rst [hunk 1]
```diff
@@ -270,7 +270,19 @@ MongoDB collection has the special field ``_id``. The connector tries to follow
...
+    SELECT CAST(_id AS VARCHAR), * FROM orders WHERE _id = ObjectId('55b151633864d6438c61a9ce');
+    
+.. code-block:: none
+
+               _id             | orderkey | orderstatus | totalprice | orderdate
+    ---------------------------+----------+-------------+------------+------------
+     55b151633864d6438c61a9ce  |        1 | bad         |       50.0 | 2015-07-23
+    (1 row)
+    
+
```
Comment (ebyhr) [inline presto-docs/src/main/sphinx/connector/mongodb.rst:285]: Remove redundant empty lines.

File: presto-docs/src/main/sphinx/connector/mongodb.rst [hunk 2]
```diff
@@ -270,7 +270,19 @@ MongoDB collection has the special field ``_id``. The connector tries to follow
 
 .. note::
 
-    Unfortunately, there is no way to represent ``_id`` fields more fancy like ``55b151633864d6438c61a9ce``.
+    One way to present ``_id`` fields more fancy like ``55b151633864d6438c61a9ce`` is to cast them to ``VARCHAR``.
+    
+.. code-block:: sql
+    SELECT CAST(_id AS VARCHAR), * FROM orders WHERE _id = ObjectId('55b151633864d6438c61a9ce');
```
Comment (ebyhr) [inline presto-docs/src/main/sphinx/connector/mongodb.rst:276]: Please add new line between L275 & L276. It's the cause of CI failure.

**Review by ebyhr — COMMENTED** [review]
Thanks for improving the documentation. Could you update the commit message as follows? We are using this [guideline](https://github.com/prestosql/presto/blob/master/DEVELOPMENT.md#format-git-commit-messages) for commit message format. 
`Document example to cast MongoDB ObjectId to VARCHAR`

**Comment by ebyhr** [comment]
@blcksrx I meant a commit (≠ PR title). Also, please submit CLA if you haven't yet sent it.

**Comment by blcksrx** [comment]
Sorry. I fixed it and I submitted the CLA about a minute ago

File: presto-docs/src/main/sphinx/connector/mongodb.rst [hunk 3]
```diff
@@ -270,7 +270,18 @@ MongoDB collection has the special field ``_id``. The connector tries to follow
 
 .. note::
 
-    Unfortunately, there is no way to represent ``_id`` fields more fancy like ``55b151633864d6438c61a9ce``.
+    One way to present ``_id`` fields more fancy like ``55b151633864d6438c61a9ce`` is to cast them to ``VARCHAR``.
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/connector/mongodb.rst:273]: How about

``` 
You can render the ``_id`` field to readable values with a cast to ``VARCHAR``:
```

And then the below sections need to be indented the same as the note paragraph .. or just remove the usage of note

**Review by ebyhr — APPROVED** [review]

**Review by mosabua — APPROVED** [review]
Looks good now imho

**Comment by martint** [comment]
@cla-bot check

**Review by electrum — APPROVED** [review]

**Comment by electrum** [comment]
Merged, thanks!

**Pull request closed without merging**